composer.json

```diff
@@ -1,43 +1,52 @@
 {
     "name": "yiisoft/db-elasticsearch",
-    "description": "Yii Framework Elasticsearch Query and ActiveRecord",
-    "keywords": ["yii", "elasticsearch", "active-record", "search", "fulltext"],
+    "description": "Elasticsearch integration and ActiveRecord for the Yii framework",
+    "keywords": [
+        "yii2",
+        "elasticsearch",
+        "active-record",
+        "search",
+        "fulltext"
+    ],
     "type": "library",
     "license": "BSD-3-Clause",
     "support": {
-        "issues": "https://github.com/yiisoft/yii-elasticsearch/issues",
+        "issues": "https://github.com/yiisoft/yii2-elasticsearch/issues",
         "forum": "http://www.yiiframework.com/forum/",
         "wiki": "http://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii",
-        "source": "https://github.com/yiisoft/yii-elasticsearch"
+        "source": "https://github.com/yiisoft/yii2-elasticsearch"
     },
-    "authors": [
-        {
-            "name": "Carsten Brandt",
-            "email": "mail@cebe.cc"
-        }
-    ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
     "require": {
-        "yiisoft/arrays": "^3.0@dev",
-        "yiisoft/di": "^3.0@dev",
-        "yiisoft/strings": "^3.0@dev",
-        "yiisoft/yii-core": "^3.0@dev",
-        "ext-curl": "*"
+        "php": "^8.1",
+        "ext-curl": "*",
+        "ext-json": "*",
+        "ext-mbstring": "*",
+        "paragonie/random_compat": ">=1",
```

Comment:
I don't think we need the compat library. randon is properly supported in new versions of PHP.